requirements.in

```diff
@@ -0,0 +1,7 @@
+awscli
+biopython
+ipython
+matplotlib
+pip-tools
+scikit-learn
+toil[aws,mesos,google,encryption,cwl]
```

Comment:
This is just requirements.txt without the version numbers.  (Assuming the latest versions of everything are ok).  

If you need to pin one of these direct dependencies, you do it here in this `.in` file and it will be taken into account when running `pip-compile`.

See https://github.com/jazzband/pip-tools for more information.